### ISSUE TITLE
Is no one making patches or cheats anymore?

### ISSUE THREAD

**Issue by Uzukii** [GoldHEN/GoldHEN#152]
Sorry if this isn't the right place, but I don't know where else to look. There haven't been any new GoldHEN cheats or patches released in a long time and I'm wondering what the deal is. New PS4 games are still coming out and getting FPKGs, but support beyond that. I'm waiting on cheat codes to play Infinite Wealth and a 60 FPS patch to play Granblue. Just to name a few.

**Comment by Scalerize21** [comment]
There are. Check Goldhen's Discord server.

**Comment by Uzukii** [comment]
> There are. Check Goldhen's Discord server.

Thanks, didn't know GoldHEN had a discord. Looking in it though, I don't see any new cheats or patches for PS4 games... At least, not for the games I want. All I saw was a link to this page:
https://github.com/GoldHEN/GoldHEN_Cheat_Repository

And honestly, I can't tell if it has any new cheats or not, but I don't see a way to download them. What I'd really like to know is, why haven't THESE been updated in over a year.
https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases
https://github.com/GoldHEN/GoldHEN_Plugins_Repository

**Comment by Scalerize21** [comment]
> > There are. Check Goldhen's Discord server.
> 
> Thanks, didn't know GoldHEN had a discord. Looking in it though, I don't see any new cheats or patches for PS4 games... At least, not for the games I want. All I saw was a link to this page: https://github.com/GoldHEN/GoldHEN_Cheat_Repository
> 
> And honestly, I can't tell if it has any new cheats or not, but I don't see a way to download them. What I'd really like to know is, why haven't THESE been updated in over a year. https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases https://github.com/GoldHEN/GoldHEN_Plugins_Repository

This is the list of recently added cheats: https://github.com/GoldHEN/GoldHEN_Cheat_Repository/commits/main/

**Comment by Uzukii** [comment]
Yes, but how do I actually download them for the PS4? It's just a log of changes, like patch notes but no links to a file.

...I really hate the way Github operates. It's so anti-user friendly.

**Comment by Scalerize21** [comment]
> Yes, but how do I actually download them for the PS4? It's just a log of changes, like patch notes but no links to a file.
> 
> ...I really hate the way Github operates. It's so anti-user friendly.

Download this, add it to your PS4, and the rest should be simple.
https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases/download/v1.1.4/IV0000-GOLD00777_00-GOLDCHEATS000PS4.pkg

**Comment by Uzukii** [comment]
> > Yes, but how do I actually download them for the PS4? It's just a log of changes, like patch notes but no links to a file.
> > ...I really hate the way Github operates. It's so anti-user friendly.
> 
> Download this, add it to your PS4, and the rest should be simple. https://github.com/GoldHEN/GoldHEN_Cheat_Manager/releases/download/v1.1.4/IV0000-GOLD00777_00-GOLDCHEATS000PS4.pkg

Okay, I think I see what this is. We're expected to download that (which I already have on my PS4 because it's an old PKG that hasn't been updated since September 2023) and update it via the internet on the PS4? That's something I don't do, as I keep my PS4 entirely offline. I was looking for a way to manually update GoldHEN Cheats offline.

**Comment by Scalerize21** [comment]
@Uzukii 
![image](https://github.com/GoldHEN/GoldHEN/assets/167388200/09c50c4e-0710-48cf-aa6c-aa56b54bf973)
Download the folders: shn, mc4, and json located here > https://github.com/GoldHEN/GoldHEN_Cheat_Repository
And ftp them to your PS4.

**Comment by Uzukii** [comment]
Okay, thanks.

**Comment by Scalerize21** [comment]
If it's solved, do close it :).
